packages/artifact/src/contracts.ts

```diff
@@ -0,0 +1,28 @@
+export interface CreateArtifactResponse {
```

Comment:
Why is everything optional in this response contract? I would expect some guaranteed fields

Reply:
VSCode must have automatically changed it after I clicked `Apply suggested fixes` or something 🤔 Definitely should not have been optional